fw-child/template/query/learn-item.php

```diff
@@ -1,25 +1,42 @@
+<?php
+// Initialize current language.
+$current_lang = 'en';
+
+if ( isset( $item['lang'] ) && in_array( $item['lang'], array( 'en', 'fr' ), true ) ) {
+	$current_lang = $item['lang'];
+}
+
+$native_excerpt = apply_filters ( 'the_content', custom_excerpt ( 20, $item['id'] ) );
+
+$excerpts = array (
+    'en' => $native_excerpt ? get_field( 'excerpt', $item['id'] ) : get_field( 'excerpt', $item['id'] ), // Overrides native excerpt with custom excerpt if present
+    'fr' => get_field( 'excerpt_fr', $item['id'] ) ? get_field( 'excerpt_fr', $item['id'] ) : $native_excerpt, // Defaults back to native excerpt if no custom FR not present
+);
```

Comment:
The logic here may need a little change:
* it seems that this code would ignore using a custom excerpt if the post has a content. It should probably be the other way around: if a custom excerpt exists, use it, else default to build one from the content
* also, on line 12, there may be an error since in both cases of the ternary operator the same content is returned.